packages/hashi/src/H5P/H5PRunner.js

```diff
@@ -547,7 +523,7 @@ export default class H5PRunner {
       if (!file) {
         return;
       }
-      const json = JSON.parse(strFromU8(file.obj));
+      const json = JSON.parse(file.toString());
```

Comment:
All of these `file.toString()` changes in this file are converting what was an object, to a string, which is the same behavior, but just in a different way than before, right?

Reply:
Yes, the the `toString` method just uses the fflate `strFromU8` method under the hood, this just allows the calling code to be agnostic about the underlying library.